Using the service plugin to enable/disable a service might results in
conflicts with MCO taking care of enablement/disablement of a systemd units.
This might potentially results in block upgrades once proper checking and
error reporting is in place in future MCO versions.

--- a/tuned/plugins/plugin_service.py
+++ b/tuned/plugins/plugin_service.py
@@ -114,10 +114,10 @@ class SystemdHandler(InitHandler):
 		cmd.execute(["systemctl", "stop", name])
 
 	def enable(self, name, runlevel):
-		cmd.execute(["systemctl", "enable", name])
+		log.warn("service plugin enablement is not supported")
 
 	def disable(self, name, runlevel):
-		cmd.execute(["systemctl", "disable", name])
+		log.warn("service plugin disablement is not supported")
 
 	def is_running(self, name):
 		(retcode, out) = cmd.execute(["systemctl", "is-active", name], no_errors = [0])
